schema.sql

```diff
@@ -28,3 +28,20 @@ ALTER TABLE animals
     ADD FOREIGN KEY (species_id) REFERENCES species (id),
     ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
   
+CREATE TABLE vets (
+    id                 INT  GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name               CHAR(30),
```

Comment:
- [ ] @Yemisirach kindly change the column type from `char` to `varchar` to better represent the sting type of the name field.